index.html

```diff
@@ -173,7 +173,45 @@ <h1 class="header">What is Technica?</h1>
             <p>At Technica, we want to challenge our hackers to step out of their comfort zone and try something new, whether that's exploring a new technology, tinkering with hardware, or coding for the very first time! At Technica, we want our hackers to learn, grow, and meet new people, so join us at Technica 2021!</p>
           </div>
           <div class="col-lg-4">
-              <img id="about-image" src="img/hackers-illustration.png" alt="hackers posing in four squares of a polaroid" width="100%" height="100%">
+            <img id="about-image" src="img/hackers-illustration.png" alt="hackers posing in four squares of a polaroid" width="100%" height="100%">
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="what-can-you-do section" id="what-can-you-do">
+      <div class="container">
+        <h1 class="header">What Can You Do At Technica?</h1>
+        <div class="slide"></div>
+        <ul class="nav nav-tabs">
+          <li>
+            <a data-toggle="tab" class="tab active show orange" href="#home">
+              <div class="title orange"><div class="tab active show radio orange" href="#home"></div>
+              Make new friends
+            </div>
+          </a>
+          </li>
+          <li>
+            <a data-toggle="tab" class="tab blue" href="#menu1"><div class="title blue">
+              <div class="tab active show radio blue" href="#home"></div>
+              Learn new skills</div></a>
+          </li>
+          <li>
+            <a data-toggle="tab" class="tab pink" href="#menu2"><div class="title pink">
+              <div class="tab active show radio pink" href="#home"></div>
+              Network</div></a>
+          </li>
+        </ul>
+      
+        <div class="tab-content">
+          <div id="home" class="tab-pane fade in active show">
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+          </div>
+          <div id="menu1" class="tab-pane fade">
+            <p>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+          </div>
+          <div id="menu2" class="tab-pane fade">
+            <p>Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam.</p>
```

Comment:
i don't see fade class in scss, applying fade-in animation to both section content & active tab on click would look sick! 💥

Reply:
Just made a quick fadeIn animation :~)